routers/projectRouter.js

```diff
@@ -0,0 +1,93 @@
+const express = require("express");
+const router = express.Router();
+
+const projects = require("../data/helpers/projectModel");
+
+router.post("/", (req, res) => {
+    const {name, description, completed } = req.body;
+    if(!name || !description || completed){
+        res.status(401).json({message: "could not add project"})
+    }
+    else if (name.length > 128){
+        res.status(401).json({message: "name must ne less then 128 characters"})
+    }
+    projects
+        .insert({name, description, completed})
+        .then(response => {
+            res.status(201).json(response)
+        })
+        .call(err => {
+            res.status(500).json({message: "error adding project"})
+        })
+})
+
+router.get("/", (req, res) => {
+    projects
+        .get()
+        .then(project => {
+            res.status(200).json(project);
+        })
+        .catch (err => {
+            res.status(500).json({message:"project not found"});
+        })
+})
+
+router.get("/:id", (req, res) => {
+    const {id} = req.params;
+    projects
+        .get(id)
+        .then(project => {
+            if(project){
+                res.json(project);
+            }
+            else {
+                res.status(404).json({message: "project does not exist"})
+            }
+        })
+        .catch (err => {
```

Comment:
[Prettier will catch these inconsistencies for you 👍 ](https://github.com/prettier/prettier-vscode)
```suggestion
        .catch(err => {
```